guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
```

Comment:
I feel like there is a mismatch between the sentence and the title. I would either change the title by using a verb other than "associating" or else mention "associating" in the intro. 

LUKS should be spelled out in full the first time.

Reply:
Spelled out LUKS. IMHO it comes clear with the third sentence; so I'd keep it as is.